include/blmc_robots/solo12.hpp

```diff
@@ -0,0 +1,397 @@
+/**
+ * \file solo12.hpp
+ * \author Julian Viereck
+ * \date 21 November 2019
+ * \copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+
+ */
+
+#pragma once
+
+#include <blmc_drivers/devices/spi_motor_board.hpp>
+#include "blmc_robots/common_header.hpp"
+#include "blmc_robots/blmc_joint_module.hpp"
+
+
+namespace blmc_robots {
+
+/**
+ * @brief Definition and drivers for the Solo12 robot.
+ * 
+ *
+ * Mapping between the DOF and driver boards + motor ports:
+ * FL_HAA - motor board 0, motor port 0, motor index 0
+ * FL_HFE - motor board 1, motor port 1, motor index 3
+ * FL_KFE - motor board 1, motor port 0, motor index 2
+ * FR_HAA - motor board 0, motor port 1, motor index 1
+ * FR_HFE - motor board 2, motor port 1, motor index 5
+ * FR_KFE - motor board 2, motor port 0, motor index 4
+ * HL_HAA - motor board 3, motor port 0, motor index 6
+ * HL_HFE - motor board 4, motor port 1, motor index 9
+ * HL_KFE - motor board 4, motor port 0, motor index 8
+ * HR_HAA - motor board 3, motor port 1, motor index 7
+ * HR_HFE - motor board 5, motor port 1, motor index 11
+ * HR_KFE - motor board 5, motor port 0, motor index 10
+ * 
+ * 
+ */
+class Solo12
+{
+public:
+
+  /**
+   * @brief Solo is the constructor of the class.
+   */
+  Solo12();
+
+  /**
+   * @brief initialize the robot by setting aligning the motors and calibrate the
+   * sensors to 0
```

Comment:
Comments need grammatical and syntax fixing (colon at end, first letter upper case). Also applies to other comments.